CONTRIBUTING.md

```diff
@@ -72,6 +75,48 @@ If you do not find the suggestion, please open a new issue and clearly document
 
 Please provide your enhancement suggestions as an issue in the repository:
 
+### Dev Environments
+
+#### Manual Setup
+
+General guidance for setting up a local development environment in Linux are provided below.
+It is strongly recommended to perform this within a virtual environment such as [conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) or [pyenv](https://akrabat.com/creating-virtual-environments-with-pyenv/):
+
+```bash
+# Checkout the repository
+git clone https://github.com/cytomining/pycytominer.git
+cd pycytominer
+# Install pycytominer in development mode along with associated tools
+bash .devcontainer/postCreateCommand.sh
+```
+
+#### Local DevContainer
+
+Instructions for setting up a local development environment using VSCode DevContainers:
+
+1. Install [VSCode](https://code.visualstudio.com/download)
+2. Install the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension
+3. Open the repository in VSCode
+4. Click on the green "Reopen in Container" button in the lower left corner of the window
+5. Wait for the container to build and install the required dependencies
+
+#### Cloud Environment (Recommended)
+
+Cloud development configurations have been set up with both [Github Codespaces](https://github.com/codespaces) and [GitPod](https://www.gitpod.io/).
+These development environments have been set up with the local copy of the repository installed in development mode along with the tools needed from `requirements-dev.txt`.
+Pre-installed git hooks will ensure that any changed code is properly formatted by required tools prior to commits.
+Using a pre-built cloud development environment is the easiest way to get started contributing to pycytominer and both Gitpod and Codespaces have generous free usage tiers.
+When you are ready to make a pull request, you can use the same cloud environment to run the full test suite and ensure that your changes pass all tests.
+These cloud environments can be launched by clicking on the following links:
```

Comment:
```suggestion
These development environments include the local copy of the repository installed in development mode along with the tools specified in `requirements-dev.txt`.
Prior to commit, pre-installed git hooks auto-format any changed code.
Using a pre-built cloud development environment is an easy way to get started contributing to pycytominer, and both Gitpod and Codespaces have generous free usage tiers.
When you are ready to make a pull request, please use the same cloud environment to run the full test suite and ensure that your changes pass all tests.
You can launch these cloud environments by clicking on the following links:
```

Minor formatting suggestions to maintain consistent tone. I don't think that I changed meaning, but please fix if you spot anything

Reply:
LGTM